packages/analytics/analytics-chart/src/utils/customColors.ts

```diff
@@ -67,3 +69,20 @@ export const lookupStatusCodeCategoryColor = (label: string) => {
 
   return { solid: color, light: color }
 }
+
+const statusCodes = Array.from({ length: 500 }, (_, index) => 100 + index)
+
+export const defaultStatusCodeColorPallette: AnalyticsChartColors = {
```

Comment:
nit: can we just make it `defaultStatusCodeColors`? 

I know things autofill in VS Code nowadays, but I still prefer shorter names; plus I have a 25% chance of spelling "pallette" correctly. And i'm pretty sure that's misspelled above too 🤣